### PR TITLE
Serving static sites

### PR DESCRIPTION
**Pull request by rainux**
Now we can serve static sites directly with HTTPS-PORTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +199,34 @@ https-portal:
...
+https-portal:
+  # ...
+  environment:
+    DOMAINS: 'hexo.example.com, octopress.example.com'
+  volumes:
+    - /data/https-portal/vhosts:/var/www/vhosts:ro
+```
+
+After HTTPS-PORTAL started, it will create corresponding sub-directories for
+each virtual host in `/data/https-portal/vhosts` directory on the host machine:
```
Comment (SteveLTN) [inline README.md]: But in the above example, you mounted the volume as `:ro`.

File: fs_overlay/opt/certs_manager/models/domain.rb [hunk 2]
```diff
@@ -30,4 +32,32 @@ def dir
       "/var/lib/https-portal/#{name}-staging/"
     end
   end
+
+  def www_root
+    "/var/www/vhosts/#{name}"
+  end
+
+  def generate_welcome_page
+    return if upstream
```
Comment (SteveLTN) [inline fs_overlay/opt/certs_manager/models/domain.rb]: upstream could be empty string in some cases.

File: README.md (hunk 1 above)
Comment (rainux) [inline README.md]: Let PORTAL create vhosts directories will make initial setup much easier.

**Comment by SteveLTN** [comment]
:shipit:

**Pull request merged**